components/desktop/pulseaudio/files/pulse_exec_attr

```diff
@@ -0,0 +1 @@
+Basic Solaris User:solaris:cmd:RO::/usr/bin/pulseaudio:privs=proc_priocntl,file_dac_read
```

Comment:
OK I did not answer your question actually.
I see privilege warnings requiring file_dac_read to start pulseaudio in user-mode, this is not related to the priority.
If you do not see it and everything goes fine for you I will just assume that my machine is cursed.